src/main/resources/dk/sdu/se_f22/sortingmodule/infrastructure/SQL-create-statement.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE querys (
+	id SERIAL PRIMARY KEY,
+	tekst VARCHAR(512) NOT NULL,
+	date_stamp TIMESTAMP NOT NULL
```

Comment:
We could maybe do a "DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP", so that "date_stamp" is always up to date, and not needed to be formatted in Java.

But maybe this date_stamp maybe could be called "created_at" or something else?

Reply:
Adding onto this block, maybe make  the `teskt` column have an english name. :D